server/services/processRepo.js

```diff
@@ -66,11 +66,11 @@ var processRepo = function(repoID) {
             }
 
             // add retire results to the DB
-            repos.findAndModify({query: {repo_id: repoID}, new: true, update: {$set: {'repo_info.retire_results': retireResults}}}, function(err, doc) {
+            repos.findAndModify({query: {repo_id: repoID}, new: true, update: {$set: {'repo_info.retire_results': JSON.stringify(retireResults)}}}, function(err) { // doc could be 2nd var
               if (err) {
                 console.log('db err: ', err);
               }
-              console.log('record updated with retireResults...: ', doc);
+              console.log('record updated with retireResults...');
```

Comment:
This was problematic because the doc returned doesn't actually have the updates incorporated changes (it's the version 'pre update'), so it doesn't represent the actual change that was just made and appears as though results haven't actually been updated.